<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CrudTurtles</class>
 <widget class="QMainWindow" name="CrudTurtles">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1375</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>782</width>
    <height>345</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1375</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Administraci√≥n de tortugas</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/Images/if_Turtle_204384.ico</normaloff>:/Images/if_Turtle_204384.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(57, 134, 137)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>1251</width>
      <height>601</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Raleway</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(206, 206, 206);
background-color: rgb(231, 231, 231);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabInsert">
     <attribute name="title">
      <string>Agregar tortuga</string>
     </attribute>
     <widget class="QGroupBox" name="gbPrimaryEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>60</y>
        <width>1121</width>
        <height>221</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="lbTurtleId">
       <property name="geometry">
        <rect>
         <x>740</x>
         <y>20</y>
         <width>111</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Raleway</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Id tortuga</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lneTurtleId">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>850</x>
         <y>20</y>
         <width>201</width>
         <height>22</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Raleway</family>
         <pointsize>11</pointsize>
        </font>
       </property>
      </widget>
      <widget class="QGroupBox" name="gbEdit">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>60</y>
         <width>981</width>
         <height>61</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="lbName">
          <property name="font">
           <font>
            <family>Raleway</family>
            <pointsize>11</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Nombre</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lneTurtleName">
          <property name="maximumSize">
           <size>
            <width>350</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbSubspcie">
          <property name="font">
           <font>
            <family>Raleway</family>
            <pointsize>11</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Subespecie</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cboSubspecie">
          <property name="minimumSize">
           <size>
            <width>175</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>250</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Raleway</family>
            <pointsize>11</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbSensor">
          <property name="font">
           <font>
            <family>Raleway</family>
            <pointsize>11</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Sensor</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cboSensor">
          <property name="minimumSize">
           <size>
            <width>175</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>250</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Raleway</family>
            <pointsize>11</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="lbZona">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>170</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Raleway</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Zona</string>
       </property>
      </widget>
      <widget class="QComboBox" name="cboZona">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>170</y>
         <width>250</width>
         <height>21</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>175</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Raleway</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pbActive">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>370</y>
        <width>131</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(120, 190, 151);</string>
      </property>
      <property name="text">
       <string>Activar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pbInsert">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>370</y>
        <width>131</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(120, 190, 151);</string>
      </property>
      <property name="text">
       <string>Agregar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pbOut">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>370</y>
        <width>131</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(120, 190, 151);</string>
      </property>
      <property name="text">
       <string>Salir</string>
      </property>
      <property name="icon">
       <iconset resource="images.qrc">
        <normaloff>:/Images/door_out.ico</normaloff>:/Images/door_out.ico</iconset>
      </property>
     </widget>
     <widget class="QLabel" name="lbTextInformation">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>340</y>
        <width>341</width>
        <height>121</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(57, 134, 137)</string>
      </property>
      <property name="text">
       <string>Es mucho m√°s sencillo llevar el contol
de las tortugas monitoreadas si se
les agrega alg√∫n nombre que las
identifique del resto.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lbTurtleImage">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>320</y>
        <width>171</width>
        <height>161</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="images2.qrc">:/turtlePNG.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabQuery">
     <attribute name="title">
      <string>Buscar tortugas</string>
     </attribute>
     <widget class="QGroupBox" name="gbPrimaryQuery">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>70</y>
        <width>1081</width>
        <height>341</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QGroupBox" name="gbSecundaryQuery">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>40</y>
         <width>891</width>
         <height>61</height>
        </rect>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="lbQuery">
          <property name="font">
           <font>
            <family>Raleway</family>
            <pointsize>11</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Busqueda</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lnQuery">
          <property name="maximumSize">
           <size>
            <width>350</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Raleway</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbCritery">
          <property name="font">
           <font>
            <family>Raleway</family>
            <pointsize>11</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Criterio</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cboCritery">
          <property name="minimumSize">
           <size>
            <width>250</width>
            <height>22</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>350</width>
            <height>22</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Raleway</family>
            <pointsize>11</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QTableView" name="tvQuery">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>130</y>
         <width>891</width>
         <height>161</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Raleway</family>
         <pointsize>8</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pbOut2">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>450</y>
        <width>131</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(120, 190, 151);</string>
      </property>
      <property name="text">
       <string>Salir</string>
      </property>
      <property name="icon">
       <iconset resource="images.qrc">
        <normaloff>:/Images/door_out.ico</normaloff>:/Images/door_out.ico</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="pbActiveQuery">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>450</y>
        <width>131</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(120, 190, 151);</string>
      </property>
      <property name="text">
       <string>Activar</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbImageTF">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>430</y>
        <width>351</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="images2.qrc">:/TunderSoft.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="lbBackground">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1375</width>
      <height>700</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>lbBackground</zorder>
   <zorder>tabWidget</zorder>
  </widget>
 </widget>
 <resources>
  <include location="images2.qrc"/>
  <include location="images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pbOut</sender>
   <signal>clicked()</signal>
   <receiver>CrudTurtles</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>662</x>
     <y>248</y>
    </hint>
    <hint type="destinationlabel">
     <x>535</x>
     <y>297</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbOut2</sender>
   <signal>clicked()</signal>
   <receiver>CrudTurtles</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>707</x>
     <y>307</y>
    </hint>
    <hint type="destinationlabel">
     <x>489</x>
     <y>346</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
